### ISSUE TITLE
Anki compatibility

### ISSUE THREAD

**Issue by gh-repo-status-check** [elzody/flashcards#2]
Dear @elzody 

Nice idea! As a medical student I'm using [anki ](https://anki-web.net) as flashcard programm. It would be nice, if I can open the anki decks in your Nextcloud App!

Regards
Human from Europe

**Comment by elzody** [comment]
Indeed, the app is not currently in a usable state, but Anki compatibility is something I want to strive for in the future. :) Glad to know someone else would be interested
